examples/OpportunityCollection.json

```diff
@@ -0,0 +1,94 @@
+{
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    0,
+                    0
+                ]
+            },
+            "properties": {
+                "start_datetime": "2023-03-01T13:00:00Z",
+                "end_datetime": "2023-03-2T15:30:00Z",
```

Comment:
The latest decision I heard was that it is more consistent to use a single `/` separated 'datetime' field.